pytorch_lightning/core/lightning.py

```diff
@@ -1166,16 +1166,44 @@ def toggle_optimizer(self, optimizer: Optimizer, optimizer_idx: int):
 
```

Comment:
maybe just use the opt_idx to get the optimizer down there... just to unify the arguments with `untoggle_optimizer`. Either way is fine.

Reply:
Sounds great ! I will do so.

Reply:
Do you plan to do it in this PR?

Reply:
@tchaton ^^

Reply:
@rohitgr7 @carmocca @tchaton please see #5775